src/python_testing/TC_MWOM_2_1.py

```diff
@@ -0,0 +1,55 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
+
+import chip.clusters as Clusters
+from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main, type_matches
+from mobly import asserts
+
+# This test requires several additional command line arguments
+# run with
+# --int-arg PIXIT_ENDPOINT:<endpoint>
+
+class TC_MWOM_2_1(MatterBaseTest):
+
+    async def read_mod_attribute_expect_success(self, endpoint, attribute):
+        cluster = Clusters.Objects.MicrowaveOvenMode
+        return await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attribute)
+
+    async def send_change_to_mode_cmd(self, newMode) -> Clusters.Objects.MicrowaveOvenMode.Commands.ChangeToModeResponse:
+        ret = await self.send_single_cmd(cmd=Clusters.Objects.MicrowaveOvenMode.Commands.ChangeToMode(newMode=newMode), endpoint=self.endpoint)
+        asserts.assert_true(type_matches(ret, Clusters.Objects.MicrowaveOvenMode.Commands.ChangeToModeResponse),
+                            "Unexpected return type for ChangeToMode")
+        return ret
+
+    @async_test_body
+    async def test_TC_MWOM_2_1(self):
+
+        asserts.assert_true('PIXIT_ENDPOINT' in self.matter_test_config.global_test_params,
+                            "PIXIT_ENDPOINT must be included on the command line in "
+                            "the --int-arg flag as PIXIT_ENDPOINT:<endpoint>")
+
+        self.endpoint = self.matter_test_config.global_test_params['PIXIT_ENDPOINT']
+
+        asserts.assert_true(self.check_pics("MWOM.S.A0000"), "MWOM.S.A0000 must be supported")
+        asserts.assert_true(self.check_pics("MWOM.S.A0001"), "MWOM.S.A0001 must be supported")
+        asserts.assert_false(self.check_pics("MWOM.S.C00.Rsp"), "MWOM.S.C00.Rsp must NOT be supported")
+        asserts.assert_false(self.check_pics("MWOM.S.C01.Tx"), "MWOM.S.C01.Tx must NOT be supported")
```

Comment:
This still needs some work, right?

Reply:
These commands have X conformance, so if the DUT supports these commands, that would be a fail.

Reply:
Yeah, but are we really verifying this here? We are basically just checking the PICS as far as I understand. So If the device actually supported these commands and the PICS was set to false, this would pass?

Reply:
OK, good point.  Thanks for that insight.